main.css

```diff
@@ -44,3 +45,7 @@ h1.site-title:before {
   text-decoration: none;
   text-transform: uppercase;
 }
+
+li {
+  margin: 10px 10px 10px 10px;
```

Comment:
Ovdje nam je dovoljno specificirati samo vertikalnu marginu jer nemamo potrebe za horizontalnom. Dakle, odgovao bi nam i ovaj CSS:

~~~ css
li {
  margin: 10px 0;
}
~~~

Međutim, sada je pravo vrijeme da se spomene i kolabiranje margina: iako smo CSS-om rekli da svaki `<li>` element treba imati i gornju i donju marginu od 10px, razmak između dva uzastopna `<li>` elementa će biti samo 10px, a ne 20px!

Naime, browseri ponekad kombinuju donju marginu gornjeg elementa s gornjom marginom donjeg elementa u jednu marginu (onu najdužu). Pošto su u ovom slučaju i gornja i donja margina jednake i ta kombinovana margina će biti jednaka objema, tj. iznosit će 10px.

Zato, najčešće, koristimo samo jednu komponentu vertikalne odnosno horizontalne margine. Mogli bismo, napirmjer, staviti svim elementima samo donju marginu sa identičnim efektom:

~~~ css
li {
  margin-bottom: 10px;
}
~~~

Više o [kolabiranju margina na MDN-u](https://developer.mozilla.org/en-US/docs/Web/CSS/margin_collapsing).